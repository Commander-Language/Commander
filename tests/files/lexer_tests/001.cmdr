five = 5;
echo "2 + 3 = $five";