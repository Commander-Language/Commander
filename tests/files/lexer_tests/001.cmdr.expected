VARIABLE 'five'
EQUALS '='
INTVAL '5'
SEMICOLON ';'
CMDSTRINGVAL 'echo'
STRINGVAL
[
STRINGLITERAL '2 + 3 = '
VARIABLE 'five'
]
SEMICOLON ';'
END_OF_FILE
